lib/rspec/rails/fixture_file_upload_support.rb

```diff
@@ -14,6 +14,8 @@ def rails_fixture_file_wrapper
         resolved_fixture_path =
           if respond_to?(:file_fixture_path) && !file_fixture_path.nil?
             file_fixture_path.to_s
+          elsif respond_to?(:fixture_paths)
+            (RSpec.configuration.fixture_paths&.first || '').to_s
```

Comment:
Is it possible that `respond_to?(:file_fixture_path)` is `false` in Rails 7.1+?

Reply:
I don't think so. This module is included in `TestCase`: https://github.com/rails/rails/blob/b72b7c35dd9506b08ed040594bbd4536139e6d3f/activesupport/lib/active_support/testing/file_fixtures.rb#L16

But it could be set to `nil` explicitly if somebody decides to do `Rspec.configure { |config| config.file_fixture_path = nil }` or `self.file_fixture_path = nil` in a test case. But in that case, I'm not sure why the wrapper should return a default value. Maybe I'm missing something here.

Reply:
Does Rails not support multiple paths here? Seems odd they would allow multiple paths for fixtures but not for files?

Reply:
It is weird indeed, but I could not find any reference to a plural method in their codebase. [Here](https://github.com/rails/rails/pull/47675#discussion_r1136329301) the author of the change mentions that it is something that they might wanna do in the future.

Reply:
Does Rails solely configure this with file_fixture_path directly?

Reply:
I think so: https://github.com/search?q=repo%3Arails%2Frails%20file_fixture_path&type=code

It defaults to `"#{Rails.root}/test/fixtures/files"` most of the time and you can overwrite that config. Rails does not use any of the configured `fixture_paths` to build the `file_fixture_path`. Althought the first of the fixture paths is `"#{Rails.root}/test/fixtures"` by default, I think that relation is not enforced.